src/authentication/index.js

```diff
@@ -88,8 +88,12 @@ function Authentication(auth0, options) {
     this.baseOptions._sendTelemetry === false
       ? this.baseOptions._sendTelemetry
       : true;
+  
 
-  this.baseOptions.rootUrl = 'https://' + this.baseOptions.domain;
+    
+  this.baseOptions.rootUrl = (this.baseOptions.domain && this.baseOptions.domain.toLowerCase().startsWith('http'))
```

Comment:
We should not use `startsWith`, that is not supported in IE. Please use `indexOf === 0`.

Reply:
Wow didn't know that, thanks, fixed!